test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class PagesControllerTest < ActionDispatch::IntegrationTest
+  test 'should get hello' do
+    get pages_hello_url
+    assert_response :success
+  end
+end
```

Comment:
- [ ] Kindly address issues mentioned by the previous reviewer. You need to have as directory rspec called requests that will contain all the tests you will write(`rspec/requests`)

## [OPTIONAL]
- [ ]  Also You need to include linters in your project currently I can't see any of them